locksmith/src/payment/paymentProcessor.ts

```diff
@@ -303,6 +307,12 @@ export class PaymentProcessor {
       }
     )
 
+    if (paymentIntent.status !== 'requires_capture') {
```

Comment:
We fail before sending the blockchain transaction if we can't capture the payment on stripe.